### DISCUSSION TITLE
what version of rastervision

### DISCUSSION THREAD

**Discussion by georgeper2001** [azavea/raster-vision#2082] [General]
How do i tell what version of rastervision i am using?
i have a docker image that i run. after running that image, if i type '/opt/conda/bin/python -m pip install --upgrade pip' would that install the latest version? or should i build a new docker image?
is there a command to reveal what version of rastervision i am running with or have installed?

**Comment by AdeelH** [comment]
You can run `pip freeze | grep rastervision` to check the installed rastervision version.

Also, `/opt/conda/bin/python -m pip install --upgrade pip` updates `pip` itself and not any installed packages.
